tests/framework/common.go

```diff
@@ -242,6 +244,7 @@ func registerFlags(td *OsmTestData) {
 	flag.StringVar(&td.OsmNamespace, "OsmNamespace", utils.GetEnv("K8S_NAMESPACE", defaultOsmNamespace), "OSM Namespace")
 
 	flag.BoolVar(&td.EnableNsMetricTag, "EnableMetricsTag", defaultEnableNsMetricTag, "Enable tagging Namespaces for metrics collection")
+	flag.BoolVar(&td.DeployOnOpenShift, "deployOnOpenShift", false, "Configure tests to run on OpenShift")
```

Comment:
I'd generalize this to `linkServiceAccountsToSCC` or something like that - which would be applicable to any kind of cluster that uses SCC.